geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyIndexCommand.java

```diff
@@ -56,19 +55,16 @@
   public Result destroyIndex(
       @CliOption(key = CliStrings.DESTROY_INDEX__NAME, unspecifiedDefaultValue = "",
           help = CliStrings.DESTROY_INDEX__NAME__HELP) final String indexName,
-
       @CliOption(key = CliStrings.DESTROY_INDEX__REGION, optionContext = ConverterHint.REGION_PATH,
           help = CliStrings.DESTROY_INDEX__REGION__HELP) final String regionPath,
-
       @CliOption(key = {CliStrings.MEMBER, CliStrings.MEMBERS},
           optionContext = ConverterHint.MEMBERIDNAME,
           help = CliStrings.DESTROY_INDEX__MEMBER__HELP) final String[] memberNameOrID,
-
       @CliOption(key = {CliStrings.GROUP, CliStrings.GROUPS},
           optionContext = ConverterHint.MEMBERGROUP,
-          help = CliStrings.DESTROY_INDEX__GROUP__HELP) final String[] group) {
-
-    Result result;
+          help = CliStrings.DESTROY_INDEX__GROUP__HELP) final String[] group,
+      @CliOption(key = CliStrings.IFEXISTS, specifiedDefaultValue = "true",
+          unspecifiedDefaultValue = "false", help = CliStrings.IFEXISTS_HELP) Boolean ifExists) {
```

Comment:
since we have an unspecifiedDefaultValue, then there would always be a value passed in for this parameter, we can just declare it as boolean instead of Boolean.